app/lib/controlled_work_mappings/cw2_welsh.yml

```diff
@@ -15,9 +15,15 @@
     - name: NASS
       type: boolean_radio
       source: asylum_support?
-      yes_value: "Yyd"
+      yes_value: "Yyd" # Note that this mirrors a typo in the CW form field name
```

Comment:
I thought this was odd when I was mapping this across. I too had to remove apostrophes/rename fields etc